docs/tutorials/creating-your-first-substrate-chain/interact.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Interacting with Your Node
+---
+
+Now that your node has finished compiling, let's show you how everything works
+out of the box.
+
+## Starting Your Node
+
+Run the following commands to start your node:
+
+```bash
+cd substrate-node-template/
+# Purge chain cleans up any old data from running a `dev` node in the past
+# You will be prompted to type `y`
+./target/release/node-template purge-chain --dev
+# Run your actual node in "developer" mode
+./target/release/node-template --dev
+```
+
+You should see something like this if your node is running successfully:
+
+```bash
+$ ./target/release/node-template --dev
+
+2019-09-05 15:57:27 Running in --dev mode, RPC CORS has been disabled.
+2019-09-05 15:57:27 Substrate Node
+2019-09-05 15:57:27   version 2.0.0-b6bfc95-x86_64-macos
+2019-09-05 15:57:27   by Anonymous, 2017, 2018
+2019-09-05 15:57:27 Chain specification: Development
+2019-09-05 15:57:27 Node name: unwieldy-skate-4685
+2019-09-05 15:57:27 Roles: AUTHORITY
+2019-09-05 15:57:27 Initializing Genesis block/state (state: 0x26bd…7093, header-hash: 0xbf06…58a9)
+...
+2019-09-05 15:57:30 Imported #1 (0x9f41…e673)
+2019-09-05 15:57:32 Idle (0 peers), best: #1 (0x9f41…e673), finalized #1 (0x9f41…e673), ⬇ 0 ⬆ 0
+2019-09-05 15:57:37 Idle (0 peers), best: #1 (0x9f41…e673), finalized #1 (0x9f41…e673), ⬇ 0 ⬆ 0
+```
+
+Your blockchain is producing new blocks!
+
+## Start Your Front-End
+
+To actually interact with this local node, you need to start your
```

Comment:
This section of the tutorial encourages developers to build their runtimes alongside a UI. That's reasonable, but it's not the only way to interact with the local node and therefore isn't always immediately necessary. Some developers may find it easier to interact with the node solely through the CLI while building the runtime and then integrate a UI later.
```suggestion
To interact with the local node via the UI template, start the 
```

Reply:
I agree the wording here can be less strong since there are other options. Would want to make sure that we keep the wording "front-end template" as to not confuse with the `substrate-ui` project.